activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/TaskCreatedEventHandler.java

```diff
@@ -49,6 +48,7 @@ public void handle(CloudRuntimeEvent<?, ?> event) {
 
             
             queryTaskEntity.setProcessInstance(processInstanceEntity);
+            queryTaskEntity.setProcessDefinitionName(processInstanceEntity.getProcessDefinitionName());
         }
```

Comment:
The processDefinitionName is not present in the the internalTask entity in Activity. As temporary solution I take the processDefinitionName from the process instance. We need to discuss if it's worth adding that in Activity and if we can avoid duplications.

Reply:
I'd prefer to keep this at the query service only, so we can keep the execution database as light as possible. So I think the current solution is a fair one.